packages/datatrak-web/src/api/mutations/useLogout.ts

```diff
@@ -11,8 +11,8 @@ export const useLogout = () => {
   const queryClient = useQueryClient();
 
   return useMutation('logout', () => post('logout'), {
-    onSuccess: () => {
-      queryClient.invalidateQueries();
+    onSuccess: async () => {
+      await queryClient.resetQueries();
```

Comment:
`invalidateQueries` triggers re-fetches of the data instantly which can lead to race conditions. `removeQueries` or `clear` doesn't work because the app no longer fetches data for queries once they are removed. ResetQueries puts the queries back to their initial state so it acts like a fresh login after logout.